Test.Utility.Test/TestEnvironmentBuilderTest.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Text;
+using Xunit;
+
+namespace Messerli.Test.Utility.Test
+{
+    public class TestEnvironmentBuilderTest 
+    {
+        [Fact]
+        public void CreateTestEnvironmentBuilder()
+        {
+            var testEnvironmentBuilder = new TestEnvironmentBuilder();
+            
+            var testFile = new TestFile("file1.txt");
+            testEnvironmentBuilder.AddTestFile(testFile);
+            var testEnvironmentProvider = testEnvironmentBuilder.Build();
```

Comment:
`new TestEnvironmentBuilder()`, `AddTestFile` and `Build` can be called fluently.

Reply:
Addressed in https://github.com/messerli-informatik-ag/test-utility/pull/11/commits/c8d6ab46905a494bd91a564b0e4a71f90a2ae690